package.json

```diff
@@ -275,7 +275,7 @@
     "@types/node": "18.16.9",
     "@types/react-copy-to-clipboard": "^4.3.0",
     "@types/react-cytoscapejs": "^1.2.2",
-    "@types/react-dom": "18.2.14",
+    "@types/react-dom": "npm:types-react-dom@19.0.0-rc.1",
```

Comment:
this feals like a strange change to me...

Reply:
Me too, codemod did this